src/MediaPlayer/MediaPlayer.cs

```diff
@@ -319,6 +324,7 @@ public enum MPMediaLibraryAuthorizationStatus : nint {
 
 	[Mac (10,12,2)]
 	[iOS (10,0)]
+	[TV (10,0)]
```

Comment:
Weird, I'm sure we had an empty unclassified w/Xcode 8.3
Thanks for checking, if intro is happy then I am :)